src/scss/generic/_typography.scss

```diff
@@ -18,14 +18,10 @@ body {
   }
 }
 
-[hidden] {
```

Comment:
This is a 'polyfill' for browsers that don't support `hidden`. It shouldn't be in the way of LastPass support as regular `hidden` also sets `display: none` via the UA stylesheet. Can we leave this one in?

Reply:
I don't think there's a single actively supported browser that doesn't support hidden? It goes all the way back to IE11.

Reply:
I see what you mean, but people get this page served through various weird browsers, like super old Webkit-based in-app browsers (for that reason we also have a localStorage polyfill).

Reply:
If you want, but WebKit has supported it since 2011 on macOS and 2012 on iOS.  :)

Reply:
It’s peace of mind with just one line of code :-) But maybe no need since it’s been in Webkit so long and we have close to 0 users of IE, let alone old IE.